tools/check_test.js

```diff
@@ -30,6 +30,12 @@ function Driver() {
     fail: 0,
     skip: 0,
     timeout: 0,
+    json: {
+      bin: { text:0, total:0, data:0, bss:0, rodata:0 },
```

Comment:
@daeyeon by issue with design I was refering to any discussion about requirements, what will be the design etc.
I understand that I don't need to worry about making json test results for now.